tools/benchmark/benchmark_html_strip.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+"""Benchmark mediawords.util.html.html_strip()."""
```

Comment:
is this intended to be checked in?  benchmark suites are great – we used https://github.com/airspeed-velocity/asv for a project, and it has been quite useful (see the [results here](http://pandas.pydata.org/speed/pymc3/))

Reply:
looks helpful. will check out.  thanks!